CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ It diverges in the following ways:
 - (disabled by default) Dialog prompting users to manually upgrade the app
 - Firefox Account sign in and Send Tab is available to release users
 
+### Fixed
+- Toasts for pinning/unpinning and desktop mode (#2864)
```

Comment:
nit: what's fixed about the toasts? With this wording, it's unclear what was ever broken (i.e. they don't appear).